src/commands/commit/apikey.ts

```diff
@@ -0,0 +1,42 @@
+import axios from 'axios'
+
+export class ApiKeyValidator {
+  public apiKey: string | undefined
+  public datadogSite: string
+
+  private isValid?: boolean
+
+  constructor(apiKey: string | undefined, datadogSite: string) {
+    this.apiKey = apiKey
+    this.datadogSite = datadogSite
+  }
```

Comment:
Nit: Typescript has some sugar to pass arguments as attributes to a class

```suggestion
  private isValid?: boolean

  constructor(public apiKey: string | undefined, public datadogSite: string) {}
```